frontend/src/pages/ForgotPassword.vue

```diff
@@ -66,15 +88,17 @@ export default {
           },
         })
         .then(() => {
-          this.$router.push('/thx/forgotPassword')
+          this.showPageMessage = true
+          this.success = true
         })
         .catch(() => {
-          this.toastError(this.$t('error.email-already-sent'))
-          this.$router.push('/thx/forgotPassword')
+          this.showPageMessage = true
+          this.success = false
         })
     },
   },
   created() {
+    // Wolle: what shall happen here? change needed?
```

Comment:
We reuse the form for Forget Password for initially setting the password as well. This is done via a different URL provided after registering, but is essentially the same process.
I would suggest to keep this as it is for now. Later on this might be worth a refactor aswell: Have two pages and reuse the same password set-component on both.

Reply:
Supi 🤗